### PR TITLE
Add plural for "pieces of a book"

### PR DESCRIPTION
**Pull request by Qrox**
#### Summary
None

### CODE REVIEW

**Comment by BrettDong** [comment]
But I don’t think characters can tell whether a scrap of papers is from a book or different books 🤣

**Comment by Qrox** [comment]
Yeah, I'm just trying to make it grammatical, otherwise it becomes `piece of a books`. Will `piece of a book / pieces of books` be better? Although I think the item is always supposed to be multiple pieces, so maybe always `pieces of books`?

**Comment by BrettDong** [comment]
I prefer `str_sp` on this.

File: data/json/items/book/abstract.json [hunk 1]
```diff
@@ -390,7 +390,7 @@
     "category": "books",
     "symbol": ",",
     "color": "white",
-    "name": { "str": "pieces of a book" },
+    "name": { "str": "pieces of a book", "str_pl": "pieces of books" },
```
Comment (Qrox) [inline data/json/items/book/abstract.json:393]:
```suggestion
    "name": { "str_sp": "pieces of books" },
```

**Pull request merged**